index.html

```diff
@@ -1,28 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    <header class="menu-wrap">
-        <nav class="nav-bar">
-            <a href="#" class="brand">My Logo</a>
-            <div class="hamburger">
-              <span class="bar"></span>
-              <span class="bar"></span>
-              <span class="bar"></span>
-            </div>
-        </nav>
-      </header>
-   <section class="wrapper">
-     <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
-     <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
-      <a href="#" class="connect">LET'S CONNECT</a><br>
-      <div class="socials">
+  <header class="menu-wrap">
+    <nav class="nav-bar">
+      <a href="#" class="brand">My Logo</a>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div>
+        <ul class="menu">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="wrapper">
+    <h1 class="title">I'm Jesse Glad to see you!</h1>
+    <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+    </p><br>
```

Comment:
- [ OPTIONAL] Nice Design but there is still something to address, I noticed that your heading and paragraph should be on the same line as we can see that on Figma design, kindly fix this for your project to look more attractive :wink: 
Your Design 
![Screenshot from 2023-02-25 19-29-33](https://user-images.githubusercontent.com/88894525/221371225-61ccbedd-9295-4890-bc98-8bf3ae325e6a.png)

Figma Design 
![Screenshot from 2023-02-25 19-28-21](https://user-images.githubusercontent.com/88894525/221371238-765ea3be-4ad2-4f86-9271-52ce316580da.png)